bitski/src/lib.rs

```diff
@@ -51,9 +51,10 @@ impl Bitski {
     }
 
     pub fn from_env() -> Result<Self, Error> {
-        let client_id = std::env::var("API_KEY").or_else(|_| std::env::var("CLIENT_ID"))?;
-        let credential_id = std::env::var("CREDENTIAL_ID");
-        let credential_secret = std::env::var("CREDENTIAL_SECRET");
+        let client_id =
+            std::env::var("BITSKI_CLIENT_ID").or_else(|_| std::env::var("BITSKI_CLIENT_ID"))?;
```

Comment:
I don't think this is right?

Reply:
Yeah, should be `BITSKI_API_KEY` and `BITSKI_CLIENT_ID`.